scripts/upload-words.js

```diff
@@ -21,61 +21,88 @@ let repositoryStartingFolder = "./words";
 
 let allWords = fs.readdirSync(repositoryStartingFolder);
 
+const findFileLocation = (fileLocation) => {
+  if (fs.existsSync(fileLocation)) {
+    return fileLocation;
+  }
```

Comment:
I guess technically, but I like this better because we're explicitly saying "if the unaltered file exists we don't need to look through these edge cases" and then the catch all is just "if the file name is really messed up we'll just give up" which will produce a theoretically helpful error message in the console.